website/www/site/content/en/documentation/runners/flink.md

```diff
@@ -197,8 +197,10 @@ The optional `flink_version` option may be required as well for older versions o
 {{< paragraph class="language-portable" >}}
 Starting with Beam 2.18.0, pre-built Flink Job Service Docker images are available at Docker Hub:
 [Flink 1.8](https://hub.docker.com/r/apache/beam_flink1.8_job_server),
-[Flink 1.9](https://hub.docker.com/r/apache/beam_flink1.9_job_server).
-[Flink 1.10](https://hub.docker.com/r/apache/beam_flink1.10_job_server).
+[Flink 1.9](https://hub.docker.com/r/apache/beam_flink1.9_job_server),
```

Comment:
Looks like 1.8 and 1.9 are still listed here, not sure if that was intentional?

Reply:
It was but I think maybe having there only the maintained versions makes more sense. I will fix.